File: 3190-find-minimum-operations-to-make-all-elements-divisible-by-three/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.cpp
class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        int cnt=0;

        for(int i=0;i<nums.size();i++){
            if(nums[i]%3==0){
                continue;
            }
            if(nums[i]%3==1){
                nums[i]--;
                cnt++;
            }
            else if(nums[i]%3==2){
                nums[i]++;
                cnt++;
            }
        }

        return cnt;
    }
};

File: 3190-find-minimum-operations-to-make-all-elements-divisible-by-three/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function(nums) {
    let cnt=0;

    for(let i=0;i<nums.length;i++){
        if(nums[i]%3==0) continue;
        else if(nums[i]%3==1){
            nums[i]--;
            cnt++;
        }
        else if(nums[i]%3==2){
            nums[i]++;
            cnt++;
        }
    }
    
    return cnt;
};